content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "O3DE Visual Studio 2022 Support!"
+date: 2022-11-09
+slug: o3de-vs2022-initial-support
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)
+
+Because O3DE uses [CMake](https://www.o3de.org/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
+The backend build server infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.  
+Before the 22.10.0 release, continuous integration builds on Windows only installed the Visual Studio 2019(VS2019) Build Tools. The [v142 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset) was the only Microsoft Visual C++(MSVC) toolset validated at the time and therefore build breakages of O3DE building against the Visual Studio 2022 v143 toolset were common and took days to discover.
+
+## What's new
+
+### Build Server Updates
```

Comment:
Maybe we should call these "CI Nodes" or " CI Runner" instead of "Server"? It might make it clear to folks that this is specific to the CI/AR process?

Reply:
I'll update it to use the terminology of CI Nodes for the Continuous Integration